Description: Fix cleanup rule in Makefile
 also remove fsoframework-2.0.vapi and fsoframework.h
Author: Rico Rommel <rico@bierrommel.de>
--- a/fsoframework/Makefile.am
+++ b/fsoframework/Makefile.am
@@ -74,6 +74,8 @@
 CLEANFILES = \
     $(BUILT_SOURCES) \
     $(libfsoframework_la_SOURCES) \
+    fsoframework-2.0.vapi \
+    fsoframework.h
     $(NULL)
 
